src/main/kubernetes/tackle-pathfinder.yaml

```diff
@@ -0,0 +1,256 @@
+    app.kubernetes.io/part-of: tackle
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: pathfinder-postgres
+      app.kubernetes.io/version: '10.6'
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: pathfinder-postgres
+        app.kubernetes.io/version: '10.6'
+    spec:
+      volumes:
+        - name: pathfinder-postgresql-data
+          persistentVolumeClaim:
+            claimName: pathfinder-postgresql
+      containers:
+        - name: postgres
+          image: 'postgres:10.6'
+          ports:
+            - containerPort: 5432
+              protocol: TCP
+          env:
+            - name: POSTGRES_USER
+              valueFrom:
+                secretKeyRef:
+                  name: pathfinder-postgresql
+                  key: database-user
+            - name: POSTGRES_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: pathfinder-postgresql
+                  key: database-password
+            - name: POSTGRES_DB
+              valueFrom:
+                secretKeyRef:
+                  name: pathfinder-postgresql
+                  key: database-name
+          resources: {}
+          livenessProbe:
+            exec:
+              command:
+                - /bin/sh
+                - "-c"
+                - |
+                  psql -U $POSTGRES_USER -d $POSTGRES_DB -c 'SELECT 1'
+            initialDelaySeconds: 60
+            timeoutSeconds: 10
+            periodSeconds: 10
+            successThreshold: 1
+            failureThreshold: 3
+          readinessProbe:
+            exec:
+              command:
+                - /bin/sh
+                - "-c"
+                - |
+                  psql -U $POSTGRES_USER -d $POSTGRES_DB -c 'SELECT 1'
+            initialDelaySeconds: 10
+            timeoutSeconds: 1
+            periodSeconds: 10
+            successThreshold: 1
+            failureThreshold: 3
+          terminationMessagePath: /dev/termination-log
+          terminationMessagePolicy: File
+          imagePullPolicy: IfNotPresent
+          volumeMounts:
+            - name: pathfinder-postgresql-data
+              mountPath: /var/lib/postgresql
+          securityContext:
+            privileged: false
+      restartPolicy: Always
+      terminationGracePeriodSeconds: 30
+      dnsPolicy: ClusterFirst
+      securityContext: {}
+      schedulerName: default-scheduler
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 25%
+      maxSurge: 25%
+  revisionHistoryLimit: 10
+  progressDeadlineSeconds: 600
+---
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    prometheus.io/port: "8080"
+    prometheus.io/scheme: http
+    prometheus.io/scrape: "true"
+    prometheus.io/path: /q/metrics
+  labels:
+    app.kubernetes.io/name: tackle-pathfinder
+    app.kubernetes.io/part-of: tackle
+    app.kubernetes.io/version: 0.0.1-SNAPSHOT-native
```

Comment:
This is more a query for my own understanding than a comment on the code: if we always use the native image for tackle-pathfinder, why do we provide Dockerfiles for running non-native image ( ie JVM) containers?

Reply:
Good point.
Basically it's Quarkus who generates these files using the code.quarkus.io web.

In fact those Docker files are not needed as we use JIB to create the image containers. I will remove them.